src/postgraphql/withPostGraphQLContext.js

```diff
@@ -0,0 +1,45 @@
+import { $$pgClient } from '../postgres/inventory/pgClientFromContext'
+import debugPgClient from './http/debugPgClient'
+import setupPgClientTransaction from './setupPgClientTransaction'
+
+/*
+
+This is intended to be called similar to this:
+
+    const result = await withPostGraphQLContext(options, { pgPool, jwtToken }, async context => (
+      await graphql(
+        schema,
+        query,
+        null,
+        {  ...context },
+        variables,
+        operationName,
+      )
+    ))
+
+*/
+
+export default async function withPostGraphQLContext(options, {pgPool, jwtToken}, functionToRun) {
```

Comment:
I think

```js
    withPostGraphQLContext({ ...options, pgPool, jwtToken }, async context => (
      await graphql(
        schema,
        query,
        null,
        {  ...context },
        variables,
        operationName,
      )
    ))
```

makes more sense than

```js
    runPostGraphQLInContext({ ...options, pgPool, jwtToken }, async context => (
      await graphql(
        schema,
        query,
        null,
        {  ...context },
        variables,
        operationName,
      )
    ))
```

I could see instead `runInPostGraphQLContext` since you're running a method in a postgraphql context; however I think `withPostGraphQLContext` is probably fine.